apps/sensenet/src/components/dialogs/permission-editor/permission-editor-members.tsx

```diff
@@ -0,0 +1,127 @@
+import { reactControlMapper } from '@sensenet/controls-react'
```

Comment:
it is almost the same exact code as in reference-content-list, we should create a new common component, which is customizable with props for the unique requirements. or make reference-content-list customizable for changes in these case, if that possible

Reply:
ok, now i see, you tried it, could we improve that?